reutils/multimatch.go

```diff
@@ -0,0 +1,55 @@
+package reutils
+
+import (
+	"regexp"
+	"sync"
+)
+
+// MultiMatch returns a map of matching elements from a list of regular expressions (returning the first matching element)
+func MultiMatch(s string, expressions ...*regexp.Regexp) (map[string]string, int) {
+	for i, re := range expressions {
```

Comment:
the `i` should get renamed to be less confusing.
`reIndex` and `groupIndex`

Reply:
You are right. Done!